.github/CODEOWNERS

```
@@ -508,6 +508,11 @@ plaid/assets/logs/                                                  @DataDog/saa
 /gpu/*.md                                                            @DataDog/ebpf-platform @DataDog/documentation
 /gpu/manifest.json                                                   @DataDog/ebpf-platform @DataDog/agent-integrations @DataDog/documentation
 
+/watchguard_firebox/                                                  @DataDog/agent-integrations
+/watchguard_firebox/*.md                                              @DataDog/agent-integrations @DataDog/documentation
+/watchguard_firebox/manifest.json                                     @DataDog/agent-integrations @DataDog/documentation
+/watchguard_firebox/assets/logs/                                      @DataDog/agent-integrations @DataDog/documentation @DataDog/logs-backend @DataDog/logs-core
```

```suggestion
/watchguard_firebox/assets/logs/                                      @DataDog/agent-integrations @DataDog/documentation @DataDog/logs-backend @DataDog/logs-integrations-reviewers
```

I think log integrations review are handled by @DataDog/logs-integrations-reviewers instead of @DataDog/logs-core

---

Let us check this and get back to you.

---

@NouemanKHAL  We had a call with @torosmassa and he will have a look into this.

---

@NouemanKHAL you have to let the logs team know when a PR is ready for review via Jira . I took care of [this](https://datadoghq.atlassian.net/browse/LOI-420) and others by moving the ticket to ToDo, otherwise they won't review PRs.